packages/kujua-sms/views/lib/smsparser.js

```diff
@@ -201,6 +202,8 @@ exports.parse = function (def, doc) {
         return {};
     }
 
+    if(doc.message) doc.message = standardiseDigits(doc.message);
```

Comment:
I don't think we want to modify the original message?  We want to save the original contents of the message?